src/openforms/appointments/contrib/qmatic/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+from zgw_consumers.constants import APITypes
+
+
+class QmaticConfigManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().select_related("service")
+
+
+class QmaticConfig(SingletonModel):
+    """
+    Global configuration and defaults
+    """
+
+    service = models.OneToOneField(
+        "restapis.RestAPI",
+        verbose_name=_("Calendar API"),
+        on_delete=models.PROTECT,
+        related_name="+",
+        null=True,
+        help_text=_(
+            "The Qmatic Orchestra Calendar Public Appointment API service. Example: https://example.com:8443/calendar-backend/public/api/v1/"
+        ),
```

Comment:
```suggestion
        help_text=_(
            "The Qmatic Orchestra Calendar Public Appointment API service. "
            "Example: https://example.com:8443/calendar-backend/public/api/v1/"
        ),
```
Please break long strings over multiple lines (black uses 88 chars line length).